tests/automation/message_requests.spec.ts

```diff
@@ -146,7 +150,48 @@ test_Alice_1W_Bob_1W(
   },
 );
 
-// Clear all requests
-
+test_Alice_2W_Bob_1W(
+  'Block message request',
+  async ({ alice, bob, aliceWindow1, aliceWindow2, bobWindow1 }) => {
+    const testMessage = `Sender: ${alice.userName}, Receiver: ${bob.userName}`;
+    // send a message to User B from User A
+    await sendNewMessage(bobWindow1, alice.accountid, `${testMessage}`);
+    // Check the message request banner appears and click on it
+    await clickOnTestIdWithText(aliceWindow1, 'message-request-banner');
+    // Select message request from User A
+    await clickOnTestIdWithText(
+      aliceWindow1,
+      'module-conversation__user__profile-name',
+      bob.userName,
+    );
+    // Block User A
```

Comment:
maybe update those comments to be Alice and Bob?

Reply:
Good catch